managed/devops/bin/yb_platform_util_py3.py

```diff
@@ -0,0 +1,483 @@
+    response = call_api(universe_create_url, auth_uuid, universe_config)
+    universe_json = json.loads(response.read())
+    task_id = universe_json['taskUUID']
+    response = {"data": task_id, "status": "success", "error": ""}
+    print(response)
+
+
+def create_universe(base_url, customer_uuid, auth_uuid, input_file, universe_name=""):
+    """
+    Create the universe using the json and provided universe name.
+
+    :param base_url: base url of the platform back end.
+    :param customer_uuid: UUID of the customer.
+    :param auth_uuid: authentication token for the customer.
+    :param input_file: Directory of the stored universe json data.
+    :param universe_name: Name of the new universe to be created.
+    :return: None
+    """
+    configure_json = modify_universe_config(input_file, universe_name)
+    universe_config_json = post_universe_config(configure_json, base_url, customer_uuid, auth_uuid)
+    # add the missing fields in the config json.
+    universe_config_json["clusterOperation"] = "CREATE"
+    universe_config_json["currentClusterType"] = "PRIMARY"
+    # Call universe config api to get the update config data.
+    universe_config_json = post_universe_config(
+        universe_config_json,
+        base_url,
+        customer_uuid,
+        auth_uuid
+    )
+    if universe_config_json:
+        create_universe_from_config(universe_config_json, base_url, customer_uuid, auth_uuid)
+    else:
+        data = {
+            "data": 'Unable to Configure Universe for Customer %s' % customer_uuid,
+            "status": "failed",
+            "error": 'Unable to Configure Universe for Customer %s' % customer_uuid
+        }
+        handle_response(data)
+
+
+@exception_handling
+def get_task_details(task_id, base_url, customer_uuid, auth_uuid):
+    """
+    Get details of the ongoing task.
+
+    :param task_id: Task UUID.
+    :param base_url: base url of the platform back end.
+    :param customer_uuid: authentication token for the customer.
+    :param auth_uuid: authentication token for the customer.
+    :return: None
+    """
+    task_url = base_url + "/api/v1/customers/" + customer_uuid + "/tasks/" + task_id
+    response = call_api(task_url, auth_uuid)
+    universe_json = json.loads(response.read())
+    if universe_json["status"] == "Running":
+        response = {"data": int(universe_json["percent"]), "status": "success", "error": ""}
+        print(response)
+    elif universe_json["status"] == "Success":
+        response = {"data": 100, "status": "success", "error": ""}
+        print(response)
+    elif universe_json["status"] == "Failure":
+        content = {
+            "message": "{0} failed".format(universe_json["title"]),
+            "details": universe_json["details"]
+        }
+        response = {"data": content, "status": "success",
+                    "error": content}
+        print(response)
+
+
+def create_universe_config(universe_data, universe_name):
```

Comment:
Basically, we have one option in the script where the user can pass the custom name of the universe and it will overwrite the universe.  universe_name is the name required to overwrite in current universe data.